smartsim/settings/arguments/launch/alps.py

```diff
@@ -224,5 +224,5 @@ def set(self, key: str, value: str | None) -> None:
             )
             return
         if key in self._launch_args and key != self._launch_args[key]:
```

Comment:
Not technically part of this PR, but I think this should also read
```suggestion
        if key in self._launch_args and value != self._launch_args[key]:
```
?